api.py

```diff
@@ -615,12 +615,36 @@ def get_article(
     """
     Fetch an individual article record by ID.
     """
+    source = {
```

Comment:
This source:includes data structure is defined up at cs_basic_query already- it's DRYer to use that and to modify the query value, rather than redefine that whole thing here.

Reply:
Sure